src/N3DataFactory.js

```diff
@@ -273,7 +291,8 @@ export function termToId(term, nested) {
 // ## Quad constructor
 export class Quad extends Term {
   constructor(subject, predicate, object, graph) {
-    super('');
+    super(undefined);
```

Comment:
Alright, no problem.

However, I'm just curious to learn why. Am I right that you want the variable to be initialized with something so that it is visible on an API level? I have to admit that I am not that deep into JavaScript to know all the consequences. It was my reasoning that when passing `undefined` the variable might not occupy any memory as interpreters might just delete it and thus having the API behave as before.

Reply:
> It was my reasoning that when passing `undefined` the variable might not occupy any memory

That's not how JavaScript engines work today; the will still explicitly assign the field.

> However, I'm just curious to learn why. 

Mainly for explicitness. `undefined` is what happens when we don't pass anything, forget stuff, uninitialized, etc. `null` signifies a choice. Choices are good.